src/gui/src/layoutViewer.cpp

```diff
@@ -1746,6 +1759,7 @@ void LayoutViewer::drawInstanceOutlines(QPainter* painter,
 
   painter->setPen(QPen(Qt::gray, 0));
   painter->setBrush(QBrush());
+  painter->setRenderHint(QPainter::RenderHint::Antialiasing, false);
```

Comment:
I question the value of the anti-aliasing at all. This feels like a good tradeoff for a close to 4x improvement in FPS.

Reply:
I would say that if the drawing quality suffers, we should keep the anti-aliasing, since I'm sure there are other improvements that will have bigger impact without reducing the image quality.

Reply:
Anti-Aliasing isn't cheap though, it nearly doubles the cost of draw calls for a what I think is a pretty small benefit in the grand scheme of things. If we had GPU acceleration I wouldn't mention it at all, but CPU renderers just do not have the memory bandwidth to drive interactive experiences without significant tradeoffs. 

With high resolution monitors becoming so commonplace it feels like this is a luxury feature that isn't adding as much value as it used to, while detracting from interactivity which is very annoying for the user. I'm amenable to make it a view option if you think that's better.